src/pages/CommitQueue.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { PageWrapper } from "components/styles";
+import { useParams } from "react-router-dom";
+import Badge from "@leafygreen-ui/badge";
+import { uiColors } from "@leafygreen-ui/palette";
+import { useQuery } from "@apollo/react-hooks";
+import get from "lodash/get";
+import { Skeleton } from "antd";
+import { PageTitle } from "components/PageTitle";
+import { ErrorWrapper } from "components/ErrorWrapper";
+import { CommitQueueCard } from "./commitqueue/CommitQueueCard";
+
+import {
+  GET_COMMIT_QUEUE,
+  CommitQueueQuery,
+} from "gql/queries/get-commit-queue";
+
+const { gray } = uiColors;
+
+export const CommitQueue = () => {
+  const { id } = useParams();
+  const { data, loading, error } = useQuery<CommitQueueQuery>(
+    GET_COMMIT_QUEUE,
+    {
+      variables: { id: id },
+    }
+  );
+  if (loading) {
+    return <Skeleton active={true} title={true} paragraph={{ rows: 4 }} />;
+  }
```

Comment:
page title can still be displayed if data is loading. instead of blocking rendering of all the ui, pass the loading prop to `PageTitle` and conditionally render the parts of the app that require data.